neuraxle/base.py

```diff
@@ -387,7 +419,21 @@ class MetaStepMixin:
     def __init__(self, wrapped: BaseStep = None):
         self.wrapped: BaseStep = wrapped
 
-    def set_hyperparams(self, hyperparams: HyperparameterSamples) -> 'BaseStep':
+    def setup(self, step_path: str, setup_arguments: dict = None) -> BaseStep:
+        name__ = MetaStepMixin.__name__
+
+        self.wrapped.setup(
+            step_path=os.path.join(step_path, '{0}({1})'.format(self.wrapped.name, name__)),
+            setup_arguments=setup_arguments
+        )
+
+        return self
```

Comment:
We need to do `self.is_initialized = True` here before the return. Also, use the `self.name` instead of getting a new one. Also I'm unsure about the `'{0}({1})'` thing.